installer/x86_64/install.sh

```diff
@@ -412,7 +412,7 @@ demo_install_uefi_grub()
     efibootmgr --quiet --create \
         --label "$demo_volume_label" \
         --disk $blk_dev --part $uefi_part \
-        --loader "/EFI/$demo_volume_label/grubx64.efi" || {
+        --loader "/EFI/$demo_volume_label/shimx64.efi" || {
```

Comment:
does this affect non secure boot?

Reply:
Yes, but not if  we add shimx64.efi to the EFI partition. Still working on this.